docs/desktop-wpf/index.yml

```diff
@@ -0,0 +1,65 @@
+### YamlMime:Landing
+
+title: Windows Presentation Foundation (WPF) documentation
+summary: Learn about Windows Presentation Foundation, an open-source, graphical user interface for Windows.
+
+metadata:
+  title: Windows Presentation Foundation (WPF) for .NET Core documentation
+  description: Learn about Windows Presentation Foundation, an open-source, graphical user interface for Windows.
+  ms.topic: landing-page
+  ms.date: 03/25/2020
+
+# linkListType: architecture | concept | deploy | download | get-started | how-to-guide | learn | overview | quickstart | reference | sample | tutorial | video | whats-new
+
+landingContent:
+
+  # Card
+  - title: Learn about WPF
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: Windows Presentation Foundation overview
+            url: overview/index.md
+          - text: XAML in Windows Presentation Foundation
+            url: fundamentals/xaml.md
+
+  # Card
+  - title: Get started with WPF
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: Differences from .NET Framework
+            url: migration/differences-from-net-framework.md
+          - text: Migrate from Framework to Core
+            url: migration/convert-project-from-net-framework.md
+      - linkListType: tutorial
+        links:
+          - text: Create a WPF application
+            url: https://docs.microsoft.com/visualstudio/get-started/csharp/tutorial-wpf?toc=/dotnet/desktop-wpf/toc.json&bc=/dotnet/breadcrumb/toc.json
+
+  # Card
+  - title: Fundamentals - XAML
+    linkLists:
+      - linkListType: overview
```

Comment:
```suggestion
      - linkListType: concept
```

Reply:
Actually only "Define XAML resources" should go under a concept list type. Apologies.

Reply:
Thanks, I forgot to check these.